tests/unit/test_matrix.py

```diff
@@ -242,3 +243,16 @@ def it_knows_how_to_assemble_its_blocks(self, request):
             [bs._base_values, bs._subtotal_columns],
             [bs._subtotal_rows, bs._intersections],
         ]
+
+
+class Describe_SumSubtotals(object):
+    @pytest.mark.parametrize(("prop_name",), (("foo",), ("bar",)))
+    def it_returns_correct_base_values(self, _cube_result_matrix_prop_, prop_name):
+        ss = _SumSubtotals(_cube_result_matrix_prop_, prop_name)
```

Comment:
_SumSubtotals is called with a cube-result-matrix _instance_, not with a property.

```python
def it_knows_the_base_values_to_help(self, cube_result_matrix_, prop_name):
    setattr(cube_result_matrix, prop_name, [1, 2, 3])
    assert _SumSubtotals(cube_result_matrix, prop_name)._base_values == [1, 2, 3]
```

So in fact, in order for this to work, the prop-names will need to be real ones defined (but not necessarily implemented yet) on `_BaseCubeResultMatrix`.